docs/api.rst

```diff
@@ -38,8 +38,14 @@ Models
    .. function:: strcmp
 
 EVM
----
+====
 .. automodule:: manticore.platforms.evm
+API
```

Comment:
this should be named "Seth", not "API" for now, and updated when we decide on a non placeholder name